content/source/layouts/cloud.erb

```diff
@@ -22,5 +22,11 @@
 
   <% end %>
 
-  <%= yield %>
+  <% if current_page.data.page_id %>
+    <div id="<%= current_page.data.page_id %>">
```

Comment:
Ah, clever hack. 👍🏼 

Since the web platform team's gonna have to re-implement the whole site soon, possibly worth adding a comment about what we used this for — they could figure it out with git logs, but that'd save them a bit of time, and I'm expecting them to be a bit crunched.